astrosat/views.py

```diff
@@ -246,6 +253,21 @@ class DatabaseLogRecordViewSet(viewsets.ReadOnlyModelViewSet):
     filterset_class = DatabaseLogRecordFilterSet
 
 
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def feature_tracking(request):
+    """
```

Comment:
So this is a personal preference, feel free to argue against it...

This fn creates a `LogRecord`.  I would indicate that in the name of the fn.  Notice that the Class-Based-View above it is called "DatabaseLogRecordViewSet"  (And it's arguable that I should have used a `ListApiView` instead of a ViewSet there). 

So I would maybe have called this something like `create_log_records_view`.

Also, in "urls.py", the route is defined as:

```
    path("logs/tracking", feature_tracking, name="log-tracking"),
```

The standard DRF route for creating models would just be `/logs/` and the name would be "logs-create"

The counter-argument is that this should really only be used for tracking client features, and in that case I can understand your choice of names and routes.  I'm certainly willing to lose the argument for the route path (less-so for the fn name).

Nonetheless, presumably the function in the frontend will be called something like `trackFeatures` to make it clear what's going on.

Reply:
I found `log_records`, very generic and as you allude to, chose this to make it explicit what it is for. I could though easily have added a docstring to make this clear though. I prefer to name things to their purpose and not some generic create/delete/update etc, I find that too generic. How about

- Rename function to `create_log_records` (I don't see adding `_view` as useful as we already know it is a view, seems redundant to me adding `_view` to the function name.
- I would have used `log` or `tracking` as the url, but since the Viewset uses `logs`, I felt it better to put it under that, but have an addition, as I doubt I can just use `logs` either. If I'm correct, lets leave the URL `logs/tracking` so it is explicit, as well as the name, but I will definitely change the function name

Does that sound okay?

Reply:
sounds good.

Reply:
I've now done the renaming and error checking, so can now be re-reviewed when you get the chance.